test/unit/uk/gov/hmrc/bindingtariffrulingfrontend/model/PagedTest.scala

```diff
@@ -26,28 +28,130 @@ class PagedTest extends BaseSpec {
     }
 
     "calculate size" in {
-      Paged.empty.size    shouldBe 0
-      Paged(Seq("")).size shouldBe 1
+      Paged.empty.size                     shouldBe 0
+      Paged.empty(SimplePagination()).size shouldBe 0
+      Paged(Seq("")).size                  shouldBe 1
     }
 
     "calculate isEmpty" in {
-      Paged.empty.isEmpty    shouldBe true
-      Paged(Seq("")).isEmpty shouldBe false
+      Paged.empty.isEmpty                     shouldBe true
+      Paged.empty(SimplePagination()).isEmpty shouldBe true
+      Paged(Seq("")).isEmpty                  shouldBe false
     }
 
+    // scalastyle:off magic.number
```

Comment:
this could be added for whole class here as it is only using numbers for pagination